Tickets/preops 125 (#8)
lsst/rtn-011: summary.tex

\section{Summary}

Rubin Observatory is putting in place a dedicated {\it Early Science Program} to ensure high-impact science during the first year of operation of the \lsst.
This Early Science Program is motivated by the fact that in the current baseline there will be no science-ready data products released before \drone. 

> One of the four science pillars of LSST is time-domain astronomy, which is enabled by alerts on LSST detections of transient, variable, and/or moving objects. 

Reviewer: This is not quite right: the science pillar is "exploring the transient & variable sky."  All three of the of the other science pillars involve time-domain astronomy (e.g., by identifying solar system objects, or SNe for cosmology).

Reviewer: ```suggestion
Time-domain astronomy is a key component of LSST's four science pillars and is enabled by alerts on LSST detections of transient, variable, and/or moving objects. 
```

Alerts are the only data product that will be immediately available (within 60 seconds of image readout) and publicly shareable, i.e not subject to a proprietary period \citep{LSE-163},  \citep{RDO-013}. 
The worldwide community is actively preparing to process the LSST alert stream and use it to generate groundbreaking scientific results. Additionally, for many science goals, time-sensitive follow-up observations after discovery are crucial to take full advantage of the Rubin data. 

A key component of the \esp is the capability to build {\it incremental} templates from on-sky imaging as it becomes available during commissioning and the early phases of the survey. 
Such templates will be built periodically as images accumulate to allow for partial alert generation over an incomplete sky footprint.

> Where possible, templates will be built from all commissioning data before the start of year one and used to generate alerts during year one.

Reviewer 2: from all available commissioning ...

> Where possible, templates will be built from all available commissioning data before the start of year one and used to generate alerts during year one.

Reviewer: (If we are able to make incremental templates during LSSTCam commissioning I'm not certain if we'd want to replace them with DP2-derived templates later, as this sentence implies.)

Author: I didn't mean to imply that. I intended "built from all available commissioning data before the start of year one" to be agnostic to method, i.e built either invrementally or in a final DRP. How would you suggest I rephrase this?

Reviewer: I think it would be fine just to delete the sentence entirely--the other lines surrounding make the point well enough.

Author: Will follow up on this in a future release.

> How extensive these templates are at the start of full survey operations will be influenced on the overall success of commissioning.

Reviewer 2: suggest 
How extensive these templates are at the start of full survey operations will influence the overall success of commissioning.

Author: I don't understand this. Generating templates is not a goal of commissioning so how does the state of the template library at the start of operations have any bearing on the success of commissioning?

Author: I did have a typo in my original sentence that might have been confusing.  on -> by
